### PR TITLE
Add Japanese locale

### PR DESCRIPTION
**Pull request by jeandeaual**
Thanks for the great plugin.

Here is the Japanese translation.

### CODE REVIEW

File: lib/locales/ja.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+zh_hans:
```
Comment (markets) [inline lib/locales/ja.yaml:1]: This should be `ja:` :)

**Review by markets — CHANGES_REQUESTED** [review]
Hi @jeandeaual 👋🏼 and thank you so much for your contribution! Just added a minor comment

File: lib/locales/ja.yaml (hunk 1 above)
Comment (jeandeaual) [reply lib/locales/ja.yaml]: Sorry, fixed 😅

**Review by markets — APPROVED** [review]
Let's wait to CI, but ✅ 

Thanks!

**Comment by markets** [comment]
@jeandeaual released as `v0.14.0` 🚀 

https://rubygems.org/gems/jekyll-timeago/versions/0.14.0

**Pull request merged**